recipes/gatk4/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version="4.0a1.2.7.2" %}
+
+about:
+  home: https://www.broadinstitute.org/gatk/
+  license: BSD
+  summary: Genome Analysis Toolkit (GATK4), currently in alpha
+
+package:
+  name: gatk4
```

Comment:
@chapmanb why did you call this gatk4? Why not gatk? The gatk-launch script will conflict with future versions anyway.